### DISCUSSION TITLE
Addition of hardware category "RP 2040"

### DISCUSSION THREAD

**Discussion by Th3H0ff** [meshtastic/Discussions#49] [General]
Greetings
I have a thread related to RAK 11310 which has a RP 2040. My OCD would require me to post this under hardware/RP 2040.

Peace

**Comment by GUVWAF** [comment]
@rcarteraz Could you add a RP2040/RP2350 topic?

**Reply by rcarteraz** [reply]
Done! @Th3H0ff Sorry for missing that one!

**Reply by GUVWAF** [reply]
Thanks!
